### DISCUSSION TITLE
我有20几W个界面， hexo gen一开始是内存过高， 我处理好内存后， 又报同时打开的文件数过多， 我是windows， 我该如何解决？

### DISCUSSION THREAD

**Discussion by huangapple** [hexojs/hexo#5284] [Question / Answer] [answered]
我有20几W个界面， hexo gen一开始是内存过高， 我处理好内存后， 又报同时打开的文件数过多， 我是windows， 我该如何解决？

而且我搞不明白， 为什么要同时打开那么多文件？

不能一个个markdown处理吗？这个内存会有个峰值， 打开的文件数也会有峰值 ， 而不是像现在， 跟界面线性相关

**Comment by 7zMonkey** [comment, marked as answer]
其实文档中有配置最大同时生成文件数量的配置项，默认是无上限
> -c, --concurrency  最大同时生成文件的数量，默认无限制
https://hexo.io/zh-cn/docs/commands#generate
